docs/src/pages/demos/DemoAnimateHeight.js

```diff
@@ -35,12 +34,15 @@ for (let i = 0; i < 30; i += 1) {
 }
 
 class Slide4 extends PureComponent {
+  static contentType = swipeableViewsContext;
```

Comment:
Shouldn't it be `contextType` instead of `contentType` ?

Reply:
oh

Reply:
```suggestion
  static contextType = swipeableViewsContext;
```